app/templates/toolbarexample.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Toolbar</title>
+    <p>
+ <a href="{{ url_for('login') }}">
+  <img src="https://www.computerhope.com/cdn/media/logo-200-gray.png">
+
+     <img src="{{url_for('static', filename='Hermes.png')}}" align="middle" />
+ </a>
+</p>
+</head>
+<body>
+
+</body>
+</html>
```

Comment:
Templates should be wrapped around `{% block [block_name] %}` and `{% endblock %}`, instead of standalone HTML files.

Reply:
gotcha, i actually didn't mean to commit this file in.